### ISSUE TITLE
Clarify Uno.Material x Google Material Design

### ISSUE THREAD

**Issue by iurycarlos** [unoplatform/Uno.Themes#1305]
I think it is not that clear for new users the difference between Uno.Material and Material Design itself.

Here:

https://platform.uno/docs/articles/external/uno.themes/doc/material-getting-started.html 
We have:

`
The Uno.Material library is available as NuGet packages that can be added to any new or existing Uno solution. Uno Material lets you apply [Material Design 3](https://m3.material.io/) styling to your application with just a few lines of code.

`

And Here:

https://platform.uno/docs/articles/external/uno.themes/doc/material-migration.html
We have:

`
Uno.Material v3 (not to be confused with [Material Design 3](https://m3.material.io/) from Google) introduces support for [Lightweight Styling](https://platform.uno/docs/articles/external/uno.themes/doc/lightweight-styling.html) as well as some breaking changes to the default style keys for some controls. Refer to the tables below for the changes that have been made within Uno.Material.

`

Maybe would be nice to clarify to user that we have an Uno Version of the Material Design or a Design System specific for Uno, based on Material Design, or any other text which could ilustrate the real scenario.

**Comment by kazo0** [comment]
@iurycarlos yes good catch, I'll be addressing that as part of https://github.com/unoplatform/Uno.Themes/issues/1306
